be/src/olap/rowset/segment_v2/inverted_index_writer.cpp

```diff
@@ -191,7 +174,7 @@ class InvertedIndexColumnWriterImpl : public InvertedIndexColumnWriter {
         } else {
             field_config |= int(lucene::document::Field::INDEX_TOKENIZED);
         }
-        _field = _CLNEW lucene::document::Field(_field_name.c_str(), field_config);
+        _field = new lucene::document::Field(_field_name.c_str(), field_config);
```

Comment:
why _field is not unique ptr?

Reply:
field is released by index writer when destructor called